src/script/lua_api/l_object.cpp

```diff
@@ -738,6 +738,28 @@ int ObjectRef::l_get_attach(lua_State *L)
 	return 5;
 }
 
+// get_children(self)
+int ObjectRef::l_get_children(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	ObjectRef *ref = checkobject(L, 1);
+	ServerActiveObject *sao = getobject(ref);
+	if (sao == NULL)
```

Comment:
```suggestion
	if (!sao)
```

Reply:
The most clear is `if (sao == nullptr)`

Reply:
Same situation and question of the previous review

Reply:
Yes, same as before.  You can use either `nullptr` or `!`, but `NULL` is discouraged.

Reply:
I'll go with nullptr then, I think it's more readable